### DISCUSSION TITLE
Add Support for Dimming Outlets

### DISCUSSION THREAD

**Discussion by slave0ne** [iklein99/homebridge-smartthings#122] [Ideas]
The Satco S11270 dimming outlets I found are one of the only models that don't seem to require an additional (and usually expensive) controller. SmartThings can pull my Satco dimming outlets into the SmartThings app, and with this plugin I can see my outlets in Homebridge. 

Unfortunately, they are not exposed to HomeKit. With custom scenes in SmartThings, I can trigger actions to dim or brighten the lights attached to the outlets via Homebridge.

It would be great if these could somehow be exposed to HomeKit with their dimming functions enabled! I'm happy to help test however I can if this is a realistic request.

**Comment by slave0ne** [comment]
I should clarify, the outlets appear in Homebridge, but I get a failure message when trying to add the child bridge via QR code in the Home app. I am assuming it's because these outlets are unsupported...?

**Comment by slave0ne** [comment]
I had a little time to troubleshoot and I reconfigured the plugin to run natively in Homebridge, not as a child bridge. The outlets were exposed and support their full dimming operations. This is great, thank you!

**Comment by coryabooth** [comment]
Adding as a child bridge (as far as I know) should have no impact on if a device is supported or not. If a device work OOTB then it should also work in Bridged Mode. I have nine plugins, all but one are in bridged mode for performance reasons (including this one).  

I would love to be corrected on this configuration.  

Why you could not adopt in bridged mode "should" not be impacted by this specific plugin.  

Curious experience

**Reply by slave0ne** [reply]
I would agree, it's strange. My other five plugins operate in child mode, all except this one. I'm just grateful to have a working solution at this point.
